global/scripts/parallel.shlib

```diff
@@ -172,3 +178,16 @@ function par_addjob_stdin()
     par_jobinputarray+=("$jobinput")
 }
 
+#wait for jobs submitted with fslsub, to finish.
+function par_waitjobs_fslsub()
```

Comment:
If `fsl_sub -j 234,235,236 ...` works as expected, this is virtually never needed.  I'm also not fond of sticking an `fsl_sub` specific thing into the backgrounding-focused `parallel.shlib` code.  There are enough complications keeping track of backgrounded jobs, we don't need more cruft in here making it harder to read.  `fsl_sub` is also not specific to SGE, while `qstat` may be SGE-specific, so this implementation may not be portable.

There are restrictions that make it problematic to try to support `fsl_sub` using this backgrounding API - `fsl_sub` can't submit a bash function, while `parallel.shlib` can run a function backgrounded, and `parallel.shlib` also supports piping stdin to a job (we would need a helper script if we wanted to support that for fsl_sub).  There is also the issue that the backgrounding parallelism uses the `par_runjobs` call to wait for a set to complete, while `fsl_sub` is built more around explicit dependencies (and may not provide a portable way to check the job status), so I'd rather not try to combine them into a single API.